notebooks/lst_sensitivity_pyirf.ipynb

```diff
@@ -0,0 +1,1726 @@
+{
```

Comment:
<p>use fixed binning, e.g.  bins=10, range=(0.5, 10.5)  to avoid artifacts</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Reply:
<p>done, thanks!</p>